client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -4,35 +4,35 @@ import CircularProgress from '@material-ui/core/CircularProgress';
 import useStyles from './useStyles';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
 import ChatSideBanner from '../../components/ChatSideBanner/ChatSideBanner';
 import { useEffect } from 'react';
+import DashboardHeader from '../../components/DashboardHeader/DashboardHeader';
+import AddNewItem from '../../components/AddNewItem/AddNewItem';
+import ShoppingLists from '../../components/ShoppingLists/ShoppingLists';
 
+//The unused imports and the code that is commented out will be used in the future as this
+//component get completed
```

Comment:
This is not an ideal practice because it can lead to a cluttered codebase, especially if the person implementing the functionality in the future misses the commented code or needs to change it. 

It's better to only include code that's implemented and used because it's easy to just import things when you need them

Reply:
I removed all the unused code